spring-web/src/test/java/org/springframework/web/filter/RequestLoggingFilterTests.java

```diff
@@ -58,11 +146,11 @@ public void uri() throws Exception {
 		filter.doFilter(request, response, filterChain);
 
 		assertThat(filter.beforeRequestMessage).isNotNull();
-		assertThat(filter.beforeRequestMessage.contains("uri=/hotel")).isTrue();
+		assertThat(filter.beforeRequestMessage.contains("/hotel")).isTrue();
```

Comment:
```suggestion
		assertThat(filter.beforeRequestMessage).contains("/hotel");
```

This is analogous to the aforementioned `.startsWith()` change. Please apply to all affected methods.